packages/components/src/components/domainproperties/dataclasses/models.spec.ts

```diff
@@ -45,14 +44,14 @@ describe('DataClassModel', () => {
             domain: DomainDesign.create({})
         });
 
-        expect(model.getOptions()['exception']).toBeUndefined();
-        expect(model.getOptions()['domain']).toBeUndefined();
-        expect(model.getOptions()['rowId']).toBe(1);
-        expect(model.getOptions()['name']).toBe('name');
-        expect(model.getOptions()['nameExpression']).toBe('nameExpression');
-        expect(model.getOptions()['description']).toBe('description');
-        expect(model.getOptions()['sampleSet']).toBe(2);
-        expect(model.getOptions()['category']).toBe('category');
+        expect(model.options['exception']).toBeUndefined();
```

Comment:
Any time I see `foo['bar']` it most likely has weak typings and doesn't know what properties are actually available. Some times that is OK, however, in this case I think the types could be easily known since they are being translated directly from the model. I'd either make an explicit interface declaring the `options` for a domain or use TS `Pick` to declare the types passed to the `options`.

Explicit interface:
```ts
export interface DataClassDomainOptions {
    category: string;
    exception: string;
    // ... rest of properties
}

// and then the class interface can extend or the class can also implement DataClassDomainOptions
export interface IDataClassModel extends DataClassDomainOptions { ... }

// or
class DataClassModel implements IDataClassModel, DataClassDomainOptions { ... }

// with the eventual return type of "options" being DataClassDomainOptions
get options(): DataClassDomainOptions { ... }
```

Using `Pick`:
```ts
get options(): Pick<DataClassModel, 'category' | 'description' | ... | 'sampleSet'>
```